sensor/kubernetes/complianceoperator/handler_impl.go

```diff
@@ -31,6 +33,14 @@ type handlerImpl struct {
 	stopSignal concurrency.Signal
 }
 
+type scanScheduleConfiguration struct {
+	Suspend        *bool
+	Schedule       *string
+	ScanName       string
+	Request        interface{}
```

Comment:
Is it too open to have an interface{} here? It should be at least a request.

Reply:
I wonder how would I make it to accept either `ApplyComplianceScanConfigRequest_ResumeScheduledScan` or `ApplyComplianceScanConfigRequest_SuspendScheduledScan`, I do have a validateInterface function to verify the request

Reply:
We agree that this could be done in a separate PR

Reply:
we are going to address this in a separate PR, tracked in https://issues.redhat.com/browse/ROX-21657